components/PersonalSponsors.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <section class="PersonalSponsors">
+    <section-title
+      class="PersonalSponsors-Title"
+      en-title="Personal Sponsors"
+      ja-title="個人協賛"
+    />
+    <div class="PersonalSponsors-Description">
+      <p>
+        Civictech Challenge Cupでは、個人協賛を募集しております。
+        皆様からの協賛金は、全額を運営のために大切に使わせていただきます。
+      </p>
+      <p>
+        詳しい内容につきましては、下記お申込みフォームよりご確認ください。
+      </p>
+    </div>
+    <div class="PersonalSponsors-Button">
+      <v-btn
+        height="55px"
+        rounded
+        color="#170f7b"
+        outlined
+        href="https://forms.gle/rxUU9V3CQ71MpZkZ6"
```

Comment:
すんません、URL差し替え忘れてたのでそこだけ修正ください！
```suggestion
        href="https://forms.gle/4beQrCvLMGyo6waX8"
```